src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -97,6 +96,7 @@ class InfrastructureFormula<out T : VirtualUsers> private constructor(
             )
         }
 
+        val provisionedJira = provisionJira.get()
```

Comment:
Not sure how the integration test linked by Maciej in the previous PR works but perhaps we would want to copy-paste it to this PR and have something that guards this change. WDYT?

Reply:
Seems to be good hint - i need to investigate what is inside this IT